projects/js-packages/components/components/contextual-upgrade-trigger/index.tsx

```diff
@@ -1,18 +1,31 @@
 import { Icon, arrowRight } from '@wordpress/icons';
+import classnames from 'classnames';
 import Text from '../text';
 import styles from './style.module.scss';
 import { CutBaseProps } from './types';
 import type React from 'react';
 
-const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( { description, cta, onClick } ) => {
+const ContextualUpgradeTrigger: React.FC< CutBaseProps > = ( {
+	description,
+	cta,
+	onClick,
+	href,
+	target,
+	className,
+} ) => {
+	const Tag = href !== undefined ? 'a' : 'button';
+	const tagProps = Tag === 'a' ? { href, target } : { onClick };
+
 	return (
-		<button className={ styles.cut } onClick={ onClick } role="link">
+		<div className={ classnames( styles.cut, className ) }>
```

Comment:
IMO the click/redirect should happen in the entire component, did you receive something different from Design?

Reply:
It still is 👍🏼  (see my next comment)